app-modules/service-management/src/Filament/Resources/ServiceRequestTypeResource/Pages/ServiceRequestTypeEmailTemplatePage.php

```diff
@@ -110,29 +133,106 @@ public function form(Form $form): Form
     public function save(bool $shouldRedirect = true, bool $shouldSendSavedNotification = true): void
     {
         $data = $this->form->getState();
+        /** @var ServiceRequestType $record */
+        $record = $this->getRecord();
 
-        if ($this->template) {
-            $this->template->update($data);
-        } else {
-            $data['service_request_type_id'] = $this->getRecord()->id;
-            $data['type'] = $this->type;
+        if (! ServiceRequestTypeEmailTemplateTabs::active()) {
+            if ($this->template) {
+                $this->template->update($data);
+            } else {
+                $data['service_request_type_id'] = $record->getKey();
+                $data['type'] = $this->type;
 
-            $this->getRecord()?->templates()->create($data);
+                $record->templates()->create($data);
 
-            unset($this->template);
+                unset($this->template);
+            }
+        } else {
+            $filtered = array_filter(
```

Comment:
Do we need to do this? In the loop below, if we encounter a form value that is null, we can skip creating the email template, or we can use the existing value

Otherwise, if you remove the subject but keep the body, the body will not be saved instead of the body being saved and the subject remaining the same